src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -169,6 +169,9 @@ public int getDefaultCloudIndexPrefetchBufferSize() {
         return 2;
     }
 
+    @Override
+    public String getProgressMeterRecordLabel() { return "batches"; }
```

Comment:
It seems awkward to have the default batchSize be 0, which means no batching, but to have the progress units be "batches".